src/Page/Partners/Partner_.elm

```diff
@@ -286,6 +275,52 @@ viewInfo localModel { partner, events } =
         ]
 
 
+viewPartnerEvents : Model -> Data -> Html Msg
+viewPartnerEvents localModel { partner, events } =
+    let
+        eventAreaTitle =
```

Comment:
It's a little confusing having this in a let block and line 309 inline.

If you wanted to factor out more of the mark-up generation, we could split section title into a function